nbs/animation.ipynb

```diff
@@ -197,6 +198,23 @@
         "[args_d.update(a.param.values()) for a in arg_objs]\n",
         "args=AnimationArgs(**args_d)\n",
         "\n",
+        "\n",
+        "# load override settings if provided\n",
+        "if len(override_settings_path):\n",
+        "    if not os.path.exists(override_settings_path):\n",
+        "        raise ValueError(f\"Override settings file not found: {override_settings_path}\")\n",
```

Comment:
if you changed this to a warning, you could use a default location for overrides and just check to see if the file is present or not to determine if override settings are being provided.

Reply:
Since it's in the notebook it raises an Exception to prevent continuing execution. Since the user explicitly entered a path to grab settings from and it couldn't be found they need to be alerted to that so they can fix and re-run the cell. Not sure if there's another mechanism with Jupyter/Colab to stop execution of a cell in which case could do a less scary print statement and halt.

Reply:
the way I like to organize my notebooks is to generally try to separate "workhorse" cells from "parameterization" cells. This way, all the validation and any variable substitution logic happens upstream of actually starting rendering. we're getting a bit into the weeds optimizing UX for an unstable API so I'm not suggesting actually doing this, but just for the purpose of discussion: a "less scary" way you could achieve something like this would be to run a `Path.exists()` check in the cell where the user provides the `override_settings_path`. If the path doesn't exist, you can throw the exception there or even just throw a warning and replace the override path with a null value so the user can move on to the next cell if they don't care that the overrides file is missing.